src/components/user/IFXUserDetail.vue

```diff
@@ -166,7 +178,21 @@ export default {
     <IFXPageHeader>
       <template #title>{{ item.fullName || id }}</template>
       <template #actions>
-        <IFXLoginIcon v-if="item.isActive !== undefined" :isActive.sync="item.isActive" />
+        <v-row dense align="center">
+          <v-col>
+            <span style="white-space: no-wrap"><IFXLoginIcon :disabled="true" v-if="item.isActive !== undefined" :isActive.sync="item.isActive" /></span>
```

Comment:
Rather than the `style`, you can use the Vuetify helper class `text-no-wrap`

Reply:
Is that the same thing?

Reply:
Done